instrumentation/influxdb-2.4/javaagent/build.gradle.kts

```diff
@@ -0,0 +1,26 @@
+plugins {
+  id("otel.javaagent-instrumentation")
+}
+
+muzzle {
+  pass {
+    group.set("org.influxdb")
+    module.set("influxdb-java")
+    versions.set("[2.4,)")
+    assertInverse.set(true)
+  }
+}
+
+dependencies {
+  // from 2.14, it contains all methods need to instrument and can write all test cases
+  library("org.influxdb:influxdb-java:2.14")
```

Comment:
It would be nice if there was at least 1 test running on the earliest supported version to ensure ti works. Or perhaps the supported version range should be limited so that instrumentation does not apply to versions before `2.14`?